### PR TITLE
Show the quantiles and standard deviation in the numeric column

### PR DESCRIPTION
**Pull request by popcornylu**
1. Show quantiles (min, 5%, 25%, 50%, 75%, 95%, max)
2. Show std. deviation
3. Just the UI layout

image
image

** PR checklist **

- [x] Ensure you have added or ran the appropriate tests for your PR.
- [x] DCO signed

**What type of PR is this?**
New feature

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Support 5%, 25%(Q1), 50%(median), 75%(Q3), 95%

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

### CODE REVIEW

File: static_report/public/index.html [hunk 1]
```diff
@@ -1,89 +1,76 @@
...
-    />
-    <link rel="manifest" href="logo/manifest.json" />
-    <meta name="msapplication-TileColor" content="#ffffff" />
-    <meta name="msapplication-TileImage" content="/ms-icon-144x144.png" />
-    <meta name="theme-color" content="#ffffff" />
+    >
+    <link rel="manifest" href="logo/manifest.json" >
+    <meta name="msapplication-TileColor" content="#ffffff" >
+    <meta name="msapplication-TileImage" content="/ms-icon-144x144.png" >
+    <meta name="theme-color" content="#ffffff" >
```
Comment (neighborhood999) [inline static_report/public/index.html:64]: Just curious, these changes are changing by yourself (handy)? Or there are changes by scripts?

Comment (popcornylu) [reply static_report/public/index.html:64]: It's changed by script when i run `yarn build`. Is it because of different prettier settings?

Comment (popcornylu) [reply static_report/public/index.html:64]: The html is generated by this file

https://github.com/InfuseAI/piperider/blob/main/static_report/src/sdlc/core.js

Comment (neighborhood999) [reply static_report/public/index.html:64]: Got it, thanks!

**Review by wcchang1115 — APPROVED** [review]
python part LGTM

**Review by neighborhood999 — APPROVED** [review]

**Pull request merged**